scripts/update_deployments.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+'''
+scripts/update_deployments.py
+
+A script update (save) all the deplyments in mongo
+'''
+from glider_dac import app, db
+
+if __name__ == '__main__':
+    with app.app_context():
+        deployments = db.Deployment.find()
+        for d in deployments:
+            d.save()
```

Comment:
This isn't a migration, it's a script you can call to sync up/update all the deployment model.  It can be used to properly update all the old deployments `latest_file` and `latest_file_mtime` attributes. New and updating deployments are fine. This is not a script that should be run on a schedule only one-offs as necessary from within the container.